2.9.0/final.md

```diff
@@ -0,0 +1,823 @@
+- Enable cpu fp8 qconv ([#157076](https://github.com/pytorch/pytorch/pull/157076))
+
+## Release Engineering
+- Add support for CUDA 13.0 in CI/CD builds. Enable CUDA compression mode for binary size reduction for CUDA 13.0 builds ([#160956](https://github.com/pytorch/pytorch/pull/160956)) ([#161073](https://github.com/pytorch/pytorch/pull/161073)) ([#161257](https://github.com/pytorch/pytorch/pull/161257)) ([#161663](https://github.com/pytorch/pytorch/pull/161663)) ([#161316](https://github.com/pytorch/pytorch/pull/161316)) ([#160201](https://github.com/pytorch/pytorch/pull/160201)) ([#160770](https://github.com/pytorch/pytorch/pull/160770)) ([#161013](https://github.com/pytorch/pytorch/pull/161013)) ([#161916](https://github.com/pytorch/pytorch/pull/161916)) ([#162268](https://github.com/pytorch/pytorch/pull/162268)) ([#162322](https://github.com/pytorch/pytorch/pull/162322)) ([#162383](https://github.com/pytorch/pytorch/pull/162383)) ([#161833](https://github.com/pytorch/pytorch/pull/161833))
+
+- Enable CUDA 12.6, 12.8 and 13.0 support for Linux ARM64 CD builds ([#162364](https://github.com/pytorch/pytorch/pull/162364)) ([#160720](https://github.com/pytorch/pytorch/pull/160720)) ([#159481](https://github.com/pytorch/pytorch/pull/159481))
+
+- Add support for Python 3.14 in CI/CD builds ([#156889](https://github.com/pytorch/pytorch/pull/156889)) ([#157559](https://github.com/pytorch/pytorch/pull/157559)) ([#159261](https://github.com/pytorch/pytorch/pull/159261)) ([#159869](https://github.com/pytorch/pytorch/pull/159869)) ([#160593](https://github.com/pytorch/pytorch/pull/160593)) ([#160788](https://github.com/pytorch/pytorch/pull/160788)) ([#161255](https://github.com/pytorch/pytorch/pull/161255)) ([#159725](https://github.com/pytorch/pytorch/pull/159725))
+
+- Enable NVSHMEM integration ([#151261](https://github.com/pytorch/pytorch/pull/151261)) ([#153010](https://github.com/pytorch/pytorch/pull/153010)) ([#154538](https://github.com/pytorch/pytorch/pull/154538)) ([#155506](https://github.com/pytorch/pytorch/pull/155506)) ([#156685](https://github.com/pytorch/pytorch/pull/156685)) ([#158938](https://github.com/pytorch/pytorch/pull/158938)) ([#161321](https://github.com/pytorch/pytorch/pull/161321)) ([#160778](https://github.com/pytorch/pytorch/pull/160778)) ([#159907](https://github.com/pytorch/pytorch/pull/159907)) ([#160465](https://github.com/pytorch/pytorch/pull/160465))
+
+## ROCm
+- OCP Micro-scaling Format (mx-fp8/mx-fp4) Support ([#151360](https://github.com/pytorch/pytorch/pull/151360))
+- Support experimental CU carveout `torch._C._set_sm_carveout_experimental()` ([#149466](https://github.com/pytorch/pytorch/pull/149466))
+- Add FP8 rowwise support to `_scaled_grouped_mm` ([#159075](https://github.com/pytorch/pytorch/pull/159075))
+
+## XPU
+- Enable `FlexAttention` on Intel GPU ([#143553](https://github.com/pytorch/pytorch/pull/143553))
+- Enable `_int_mm` on Intel GPU ([#157769](https://github.com/pytorch/pytorch/pull/157769))
```

Comment:
Based on feedback I received from @albanD: in general, PRs only affecting private APIs (i.e. ops named beginning with an underscore) should not be visible in the release notes. This should allow you to trim a decent number of PRs from these release notes.